typescript-client/hatchet/workflow.ts

```diff
@@ -0,0 +1,30 @@
+import * as z from 'zod';
+
+import { CreateStep, CreateStepSchema } from './step';
+
+const CronConfigSchema = z.object({
+  cron: z.string(), // TODO validate cron
```

Comment:
This is validated on the backend, frontend would be nice but not strictly necessary